<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClusteringWindow</class>
 <widget class="QMainWindow" name="ClusteringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clustering</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_path_lmap">
      <property name="text">
       <string>Label map path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_path_lmap">
      <property name="text">
       <string>/Users/emoebel/serpico-fs2/github/deep-finder/unit_tests/deepfind/cluster/in/tomo9_bin1_labelmap_cropped.h5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_cradius">
      <property name="text">
       <string>Cluster radius</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_cradius">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_sthr">
      <property name="text">
       <string>Size threshold</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line_csize_thr">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbl_path_objl">
      <property name="text">
       <string>Object list path</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="line_path_objl">
      <property name="text">
       <string>out/objlist.xml</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="button_launch">
      <property name="text">
       <string>Analyze</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTextEdit" name="te_terminal_out"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>307</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
